rs/registry/canister/CHANGELOG.md

```diff
@@ -10,22 +10,25 @@ here were moved from the adjacent `unreleased_changelog.md` file.
 
 INSERT NEW RELEASES HERE
 
-## Changed
+# 2025-03-21: Proposal 135934
 
-## Deprecated
+https://dashboard.internetcomputer.org/proposal/135934
 
-## Fixed
+No "real" behavior changes. This is just a maintenance upgrade.
+
+Technically, there is a new get_chunk method, but it does not actually do anything useful yet. Watch this space.
 
-### Update the correct node operator ID in do_remove_node_directly
 
-Fix for the do_remove_node_directly function to update the correct node operator ID record.
-In the past the caller_id and the node_operator_id for the node were always the same.
-However, since #3285 the caller_id and the node_operator_id for the removed node may differ,
-and this introduces a bug in this edge case.
```

Comment:
@[daniel-wong-dfinity-org](https://github.com/daniel-wong-dfinity-org) daniel-wong-dfinity-org Pending

TBH, I do not know which proposal introduced this. I just noted that proposal 135286 was rejected, and so this should not be in CHANGELOG.md.